lab_1/main.py

```diff
@@ -92,10 +133,19 @@ def detect_language(unknown_profile: dict, profile_1: dict, profile_2: dict, top
     :param top_n: a number of the most common words
     :return: a language
     """
-    pass
+    percentage_1 = compare_profiles(unknown_profile, profile_1, top_n)
+    percentage_2 = compare_profiles(unknown_profile, profile_2, top_n)
+    if not (isinstance(percentage_1, float) and isinstance(percentage_2, float)):
+        return None
+    if percentage_1 > percentage_2:
+        return profile_1['name']
+    if percentage_1 < percentage_2:
+        return profile_2['name']
+    return [profile_1['name'], profile_2['name']].sort()[0]
```

Comment:
can you explain in writing what this line is for? (it can be ok, I want to know whether you understand or not)

Reply:
Well, this function returns the name of the language with the largest percentage of common words. If the percentage for the first language is higher, we return its name. If the percentage is higher for the second language, we return its name respectively. If both languages have the same percentage, we sort their names in alphabetical order (as required in the task) and return the first one.